shared/language-specs/languages.ts

```diff
@@ -624,10 +277,17 @@ export const languageSpecs: LanguageSpec[] = [
     },
 ]
 
+/**
+ * Returns the language spec with teh given language identifier. If no language
+ * matches is configured with the given identifier an error is thrown.
+ *
+ * @param languageID The language ID.
```

Comment:
[not blocking] I'm not fond of explicit `@param` docs that add nothing beyond what the parameter declaration itself says. They're actually _worse_ than nothing, because they add mass to the file, can get out of sync, and force the reader to explicitly pay attention to a triviality.

I'd vote for dropping this entirely, or including it only if we have something to add about the format of the string (e.g., a pointer to the canonical language ID docs or something).